EdgeMultiplay/Scripts/Util.cs

```diff
@@ -162,5 +164,58 @@ public static Quaternion ConvertFloatArrayToQuaternion(float[] floatArray, int s
                 throw new Exception("floatArray starting from the start index doesn't qualify to create a Vector3");
             }
         }
+
+        /// <summary>
+        /// Converts a Hashtable to Dictionary<string, string>
+        /// </summary>
+        /// <param name="htags">Tags Hashtable</param>
+        /// <returns>Tags of type Dictionary<string, string></returns>
+        public static Dictionary<string, string> HashtableToDictionary(Hashtable htags)
+        {
+            Dictionary<string, string> tags = new Dictionary<string, string>();
+            Debug.Log("HashtableToDictionary: " + htags + ", Count: " + htags.Count);
+            if (htags == null || htags.Count == 0)
```

Comment:
I think you can get rid of the check for `htags.Count == 0`. If htags is an empty hashtable, I think we want to return an empty dictionary instead of null (your for loop will already do that). (Same comment for the DictionaryToHashtable function)